app/components/Projects/ProjectList.tsx

```diff
@@ -0,0 +1,158 @@
+import i18next from 'i18next';
+import React, { useEffect, useState } from 'react';
+import {
+  Alert,
+  FlatList,
+  Image,
+  Linking,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import InAppBrowser from 'react-native-inappbrowser-reborn';
+import { APIConfig } from '../../actions/Config';
+import { getAllProjects } from '../../repositories/projects';
+import { Colors, Typography } from '../../styles';
+import { bugsnag } from '../../utils';
+import { LargeButton } from '../Common';
+const { protocol, cdnUrl, webAppUrl } = APIConfig;
+
+interface ProjectListProps {
+  isSelectable?: boolean;
+  onProjectPress?: any;
+  selectedProjectId?: string;
+}
+
+export default function ProjectList({
+  isSelectable,
+  onProjectPress,
+  selectedProjectId,
+}: ProjectListProps) {
+  const [projects, setProjects] = useState([]);
+
+  useEffect(() => {
+    getAllProjects().then((projectsData: any) => (projectsData ? setProjects(projectsData) : {}));
+  }, []);
+
+  const openWebView = async (link: string) => {
+    try {
+      if (await InAppBrowser.isAvailable()) {
+        await InAppBrowser.open(link);
+      } else Linking.openURL(link);
+    } catch (error) {
+      console.error(error);
+      bugsnag.notify(error);
+      Alert.alert(error.message);
+    }
+  };
+
+  return (
+    <FlatList
+      data={projects}
+      ListHeaderComponent={() => {
+        if (isSelectable) {
+          return (
+            <LargeButton
+              heading={i18next.t('label.continue_without_project')}
+              subHeading={i18next.t('label.continue_without_project_desc')}
+              onPress={() => onProjectPress(null)}
+              active={!selectedProjectId}
+            />
+          );
+        }
+        return <></>;
+      }}
+      ListFooterComponent={() => {
+        return (
+          <LargeButton
+            onPress={() => openWebView(`${protocol}://${webAppUrl}/manage-projects/add-project`)}
```

Comment:
If we add a new env with the related web app URL, we should also use it at https://github.com/Plant-for-the-Planet-org/treemapper/blob/a02ddb82bb2882d172d72d46467e979e51248efe/app/components/ProfileModal/index.js#L49-L51.

Reply:
Which URL do we want to use here. `www1.plant-for-the-planet.org` or `www.trilliontreecampaign.org`?

Reply:
We will have to add this new env variable to the GitHub actions as well.

Reply:
Even I don't the difference and which to use and I forgot to ask the same question in the group

Reply:
Confirmed, we have to use `www1.plant-for-the-planet.org`

Reply:
I added WEB_APP_URL to GitHub actions (filled with www1.plant-for-the-planet.org).